src/stylefy/core.cljs

```diff
@@ -45,17 +45,16 @@
    (assert (or (map? options) (nil? options)) (str "Options should be a map or nil, got: " (pr-str options)))
    (impl-styles/use-style! style options)))
 
-(defn use-sub-style
-  "Defines style for a component using sub-style.
-
-   The style and options are the same as you would use with use-style.
-   sub-style is the name of the sub-stale in the given style map."
-  ([style sub-style] (use-sub-style style sub-style {}))
-  ([style sub-style options]
-   (assert (or (map? style) (nil? style)) (str "Style should be a map or nil, got: " (pr-str style)))
-   (assert (or (map? options) (nil? options))
-           (str "Options should be a map or nil, got: " (pr-str options)))
-   (impl-styles/use-sub-style! style sub-style options)))
+(defn sub-style
```

Comment:
I guess there aren't many developers using this library, but I would still be careful making API breaking changes. Would it be possible not to replace use-sub-style, but instead just create this new function as new API function?